products/vpcaccess/api.yaml

```diff
@@ -18,6 +18,9 @@ versions:
   - !ruby/object:Api::Product::Version
     name: beta
     base_url: https://vpcaccess.googleapis.com/v1beta1/
+  - !ruby/object:Api::Product::Version
```

Comment:
You'll also need to remove the line that says `min_version: beta` in the resource itself.